pkg/observ/stats.go

```diff
@@ -51,6 +62,33 @@ func registerPrometheusExporter(r *mux.Router, service string) error {
 	return nil
 }
 
+func registerStatsDataDogExporter(r *mux.Router, service string) (func(), error) {
+	const op errors.Op = "observ.registerStatsDataDogExporter"
+
+	dd := datadog.NewExporter(datadog.Options{Service: service})
+	if dd == nil {
+		return nil, errors.E(op, "Failed to initialize data dog exporter")
+	}
+	view.RegisterExporter(dd)
+	return dd.Stop, nil
+}
+
+func registerStatsStackDriverExporter(r *mux.Router, projectID string) (func(), error) {
```

Comment:
Same as for DataDog. We probably don't need `mux.Router` here :slightly_smiling_face: